scripts/test-canisters.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+POCKET_IC_SERVER_VERSION=6.0.0
+POCKET_IC_SERVER_PATH="pocket-ic"
+PREVIOUS_II_WASM_PATH="internet_identity_previous.wasm.gz"
+PREVIOUS_ARCHIVE_WASM_PATH="archive_previous.wasm.gz"
+
+# Check if the script is run from the root of the project
+project_root=$(git rev-parse --show-toplevel 2>/dev/null)
```

Comment:
We normally just switch to the appropriate directory:
```
SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
cd "$SCRIPTS_DIR/.."
```

Reply:
Ah, ok, changed!